CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ After your cluster is installed, you will need to do the following:
 1. Patch the cluster version so that you can launch your own builder image:
 
 ```
-$ oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true}]}}' --type=merge
+$ oc patch clusterversion/version --patch '{"spec":{"overrides":[{"kind":"ConfigMap", "namespace":"openshift-controller-manager-operator","name":"openshift-controller-manager-images","unmanaged":true, "group": "apps/v1"}]}}' --type=merge
```

Comment:
Why does `apps/v1` work as a group? `ConfigMap` is a part of the `core` api group, not `apps`.

Reply:
the CVO's interpretation of these fields are not as literaly as one might think @adambkaplan 

But at least for me I can confirm from past usage that for core api objs like `ConfigMaps` or `Deployments`, when I override CVO managed items of those types, I have had to put `apps/v1` in there.

Full transparency though, I have not experimented with it recently to see if it takes say just `apps`, but I can say as recently as yesterday `apps/v1` worked for me as well as what @danielkucera saw.